qbraid_algorithms/bernstein_vazirani/bernvaz.py

```diff
@@ -28,7 +28,7 @@
 from qbraid_algorithms.utils import _prep_qasm_file
 
 
-def load_program(bitstring: Union[str, list[int]]) -> QasmModule:
+def generate_program(bitstring: Union[str, list[int]]) -> QasmModule:
```

Comment:
Change to `bitstring (Union[str, list[int]]): The hidden bitstring `s` as a string of '0's and '1's` as the 1s line is coming in new line - 

<img width="1384" height="528" alt="Image" src="https://github.com/user-attachments/assets/df8ee406-8731-4898-84a1-ebf42be1208f" />

Reply:
Also, update the Returns to just mention `Pyqasm module` once